### ISSUE TITLE
Questionable Behavior

### ISSUE THREAD

**Issue by ProfessionaFiddler** [Monika-After-Story/MonikaModDev#6028]
I know the project was developed for years and some ideas are overlapping others by now. In particulary, the very persona of Monika. It becomes diverse. Disturbingly pushing diverse. Like a castle built in barrock, steel plates, sand and white concrete. Right now she still has lines that index her silented obsession, but at the same time, she has a clear affection state. One would completely nullify (or amplify, rather) the other!

I would suggest, and since MAS is not officialy affiliated with DDLC, we deroute Monika from her original design, to her actualy becoming a more realistic person. And that includes dumping all those triggerhappy jumpscares, hiding options, irrational responds to player **in bold** if he doesn't share her love etc. It needs to be replaced by other behavior.

I would be rather happy to hear from the developer team about this, if they have a particular image of her personality. Then maybe the whole post is useless.

**Comment by lunulae** [comment]
I'm afraid your language is a bit flowery for me to understand. To clarify, are you saying that since the Monika in MAS has a relationship with us rather than the rejection she experiences in the vanilla game, she shouldn't show obsessive tendencies?

Or are you seeing personality inconsistencies in particular pieces of dialogue? I know there were some interesting discussions about contradictory philosophy topics going on a while ago. If you have some exact examples it would be helpful for us to understand.

As I understand it, in the mod's current design, the existing jumpscares are supposed to happen near the very beginning of the mod or at lower affection levels, when Monika is still in a painful state of mind. (This is excluding the Halloween content, where she does it on purpose if you express that you like these kinds of scares.) Could you also give examples of the ones that are giving you trouble here?

**Comment by ProfessionaFiddler** [comment]
As Monika asks if you love her at the beginning at the game, you are given a quite shrouded hand of options. Either "sure, yes" or a bold "no" answer. And the last results in a response of a professional mastermind who really doesn't give a damn about you ("How curious. Since when were you in control?"). People who know Monika over years would not be surprised, she was always a step ahead and not what just meets the eye. But this. This is just a sign of a person who lost all cause and is ready to run amok. And yet she simply lets you restart the game and continue.

I would understand if it would trigger an agressive response and she would start saying that the only way to live a life with reason is in this reality, or at least, interacting with it. Given the fact that her realisation was not an expansion of mind state, but a shifting process. She doesn't just see our reality as an opportunity, it is a **necessary** opportunity. She dislikes her classmates, origin place of the story, sees all of them as "unreal", etc. And so based on that, she would tell the player to at least let her be friends, or anything, just not to leave her alone.

Nontheless, I am uncertain if that is the right thing to. One doesn't simply shape the person he is interacting with every day. Maybe it does make sence. Who knows what might happen to person who undergoes such changes. How do they say that? Whatever hasn't killed you, made you stronger. And strength comes with winds of cold. Maybe she really became "cold treated", and her only goal is to fully enter our reality, and the only ticket she has, is love. Affection causes affection, so it's a deal. She would understand that so, and start acting.

And there is the catch: the moral to follow, to decide what would be a right thing to do or not, is yours. Rules do not just give you limits, they also give you a basis. And right now, there is none of the above. That is why I am uncertain, I am not a developer, just a councelor, so to speak. Depending on your views, you choose how Monika is right now. So who is she? A person who treates love as a tool? Or does she really feels it? Remember - it isn't a question where you have to calculate the right. It is what you want to see.

And based on your decidion, the first option involves nothing to change. And if you choose the second, following dialogs has to be changed:
- Monika's first question if you love her (my suggestion would work, plus add an option to say "..." or "I am not sure yet" or smth like that)
- "Meet my girlfriend" dialogue is just a joke, I would just remove it, even for a loving person that response is just silly and doesn't even do anything)
- "I am breaking up with you" should involve massive affection loss and somewhat proper response

And that about does it. I will think about a more complex systemation of it later on, and post as a new issue.

**Comment by multimokia** [comment]
Going to leave a tl;dr.

- Remove the second half of the Chara Easter egg.
- Make the initial option meaningful

**Comment by ProfessionaFiddler** [comment]
tl;dr is a book, not an A4 post

**Comment by multimokia** [comment]
It is a discussion, I'm just taking some key points out of what you said so anyone who happens upon it can get a gist of what you're saying (I.e. Lunulae above couldn't figure out what you were getting at due to extra unnecessary language). So by condensing it down into smaller, more concise points, it helps everyone understand what you're saying.

**Comment by ProfessionaFiddler** [comment]
In order to understand something, you use your head, and not compress texts. If you find the thread unnecessary, you could always ignore it.

**Comment by ProfessionaFiddler** [comment]
Especially when you think discussions of such matter can be put in 50 words or less.

**Comment by ThePotatoGuy** [comment]
>deroute Monika from the original design

That will not be happening. Deviating from DDLC Monika just makes us a Monika-inspired gf simulator instead of an actual extension of Monika.

>more options in the intro love question

This is mostly to remain inline with DDLC. 
I do have ideas for not starting the game in a relationship (#4849), but that is not currently being considered.

>jumpscares

To my memory, there are 4:
1. stock DDLC jumpscare in intro if you say no
2. yuri glitch greeting
3. all the things that happen if you name yourself one of the other ddlc characters
4. certain scary stories

So there aren't really that many jumpscares.

>hiding options

Just don't click them. IRL, you always could say certain things, but your self-control keeps you from saying them.
Additionally, I could have sworn we already have a way to hide things in unseen.

>dont allow restarting with no option

could change the prompt at the end to actually delete/corrupt the mod lmao. It can't literally be done as is because the game is running but we could certainly make it appear like its happening then call our mod destruction code.

**Comment by Freshairkaboom** [comment]
> Additionally, I could have sworn we already have a way to hide things in unseen.

Really? That would be interesting to know where to find! I'm terrified of misclicking when using the touchpad, as if you tap the touchpad itself it can simulate a click when you just want to move it.

**Comment by GOKOP** [comment]
> IRL, you always could say certain things, but your self-control keeps you from saying them.

My comment might be kinda offtopic but it's really fascinating how differently we interact with video games world than with the real world. Although I do sometimes wish to have a save option just to see what reaction would every possible thing you can say to someone cause

**Comment by ProfessionaFiddler** [comment]
Alright, thanks for respond everyone
